lab/login/style.css

```
@@ -86,6 +86,19 @@
   -webkit-text-fill-color: unset;
 }
 
+.name-lord.cat::after {
+  content: "Blåhaj Collector";
+  border: solid;
+  border-image: linear-gradient(to right, #c5a4ed, #a4a5ed 100%);
+  padding: 0.1em 0.2em;
+  margin: 0.35em;
+  font-size: 0.6em;
+  position: relative;
+  top: -0.25em;
+  font-weight: normal;
+  background: linear-gradient(to right, #c5a4ed, #a4a5ed 100%);
+  background-clip: text;
+}
```

try this

```suggestion

.name-lord.cat::after {
  content: "Blåhaj Collector";
  border-width: 1px;
  border-style: solid;
  border-image: linear-gradient(to right, #c5a4ed, #a4a5ed 100%) 1;
  padding: 0.1em 0.2em;
  margin: 0.35em;
  font-size: 0.6em;
  position: relative;
  top: -0.25em;
  font-weight: normal;
  background: linear-gradient(to right, #c5a4ed, #a4a5ed 100%);
  background-clip: text;
}

.at-lord.cat::after {
  all: unset;
}
```

- added missing number to end of `border-image`
- split `border` into `border-width` and `border-style`
- removed the tag from mentions, because its long